src/encoded/static/scss/encoded/modules/_facet.scss

```diff
@@ -79,6 +79,14 @@
     overflow: hidden;
 }
 
+.box.facets {
+    .horizontal {
+        .facet {
+            border-bottom: 0;
+        }
+    }
+}
+
```

Comment:
Could this border-bottom be instead added to just after line 22?

Reply:
Yes, thanks!